api/src/main/java/eu/darkbot/api/managers/QuestAPI.java

```diff
@@ -0,0 +1,194 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+/**
+ * Provides access to data for the Quests in-game.
+ */
+public interface QuestAPI extends API.Singleton {
+
+    /**
+     * @return The quest that is currently active and in view.
+     *         Even if there are several accepted quests, only the selected one is
+     *         returned.
+     */
+    @Nullable Quest getCurrentQuestDisplayed();
+
+    /**
+     * @return The quest that is currently selected in the QuestGiver
+     */
+    @Nullable Quest getCurrentQuestSeleted();
+
+    /**
+     * @return Returns additional information on the selected quest
+     */
+    @Nullable QuestListItem getCurrentQuestInfoSeleted();
+
+    /**
+     * @return Returns a list of current quests, only has data when QuestGiver is
+     *         opened
+     */
+    @Nullable List<QuestListItem> getCurrestQuests();
+
+    /**
+     * @return Returns true if the quest giver is open
+     */
+    boolean isVisibleQuestGiver();
```

Comment:
```suggestion
    boolean isQuestGiverVisible();
```

Is this if it's visible (ie: you're in range of the quest giver) or if the quest giver gui is open?
If it's the latter, maybe rename to `isQuestGiverOpen()`, as it'd better reflect it